packages/help-center/src/components/help-center-contact-page.tsx

```diff
@@ -80,7 +80,7 @@ export const HelpCenterContactPage: React.FC = () => {
 
 		if ( isLanguageSupported ) {
 			const language = getLanguage( locale )?.name;
-			return language
+			return language && hasTranslation( `Email (${ language })` )
```

Comment:
I think `hasTranslation` should get whatever `__` gets. ie. `Email (%s)`.

Reply:
Yes that's correct. 
Thanks for catching that.
Fixed this with the latest commit.